src/controllers/login/log-in-controller.ts

```diff
@@ -1,12 +1,24 @@
 import { ICredential, IToken } from "../../models/login";
 import { UserService } from "../../services/users-service";
+import { configureValidator } from "../../utils/configureValidator";
+import { errorHandler } from "../../utils/error-handler";
+import { validateFieldEmail } from "../../utils/validateFieldEmail";
+import { validateFieldRequired } from "../../utils/validateFieldRequired";
 
 class LogInController {
   constructor(private userService: UserService) {
-    const signInButton = <HTMLButtonElement>(
-      document.getElementById("sign-In-button")
+    // const signInButton = <HTMLButtonElement>(
```

Comment:
remove commented code, this code do not run at run time, is garbage on the sources.